site/docs/components/avatar/accessibility.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Avatar
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practices
+
+Decorative avatars (avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an avatar for decorative purposes, pass `aria-hidden=”true”` to hide its label from the screen reader.
+
+If an alternative icon is used inside an avatar, the icon's `aria-label` should be descriptive in relation to the avatar context, not the icon context. For example, when using `<BuildingSolidIcon/>` to represent a company `aria-label` should read "Company profile" rather than the icon's default label "building solid".
```

Comment:
```suggestion
If an alternative icon is used inside an Avatar, the icon's `aria-label` should be descriptive in relation to the Avatar context, not the icon context. For example, when using `<BuildingSolidIcon/>` to represent a company `aria-label` should read "Company profile" rather than the icon's default label "building solid".
```